.github/ISSUE_TEMPLATE/001_bug_report.yml

```diff
@@ -0,0 +1,81 @@
+name: Bug Report
+description: File a bug report.
+labels: ["bug"]
+type: "Bug"
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: Describe the bug
+      description: |
+        A clear and concise description of what the bug is.
+
+      placeholder: |
+        Please also mention any information which could help others to understand
+        the problem you're facing:
+          - What target platform are you using?
+          - What have you tried to diagnose or workaround this issue?
+          - Is this a regression? If yes, have you been able to "git bisect" it to a
+            specific commit?
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to reproduce
+      description: |
+        Steps to reproduce the behavior.
+
+      placeholder: |
+        Steps to reproduce the behavior:
+          1. mkdir build; cd build
+          2. cmake -DBOARD=board\_xyz
+          3. make
+          4. See error
+    validations:
+      required: true
```

Comment:
I kind of see why this is required, but if the bug is e.g. some invalid documentation or other bugs that aren't from build or runtime or perhaps some missing functionality w.r.t. some spec compliance, then there's no way to reproduce. In those cases I usually end up writing `N/A` (which of course can still be done here, but it does imply that this field may not be absolutely necessary)